quantities/tests/test_umath.py

```diff
@@ -226,6 +226,8 @@ def test_arctan2(self):
             np.arctan2(3*pq.V, 3*pq.V),
             np.radians(45)*pq.dimensionless
             )
+        # NumPy <1.21 raises ValueError
+        # NumPy >=1.21 raises TypeError
         self.assertRaises((TypeError, ValueError), np.arctan2, (1*pq.m, 1*pq.m))
```

Comment:
This commit could be squashed with/into the parent commit, because while reviewing the parent commit I thought that a comment in the code about the numpy versions would be helpful. The better place for the comment is IMO the commit where support for newer numpy versions is added.

Reply:
Done!